utils/db.ts

```diff
@@ -1,4 +1,7 @@
 // Copyright 2023-2024 the Deno authors. All rights reserved. MIT license.
+
+/// <reference lib="deno.unstable" />
```

Comment:
@kt3k shouldn't `"unstable": ["kv"]` within `deno.json` fix this?

Reply:
Weirdly it didn't. I see there is `--unstable-kv` option provided when running `deno task test`, but same problem. I think it could be some deno issue. I have been able to make it work only with the `reference ... ` comment.

Reply:
Yeah, this seems like a runtime issue.